### PR TITLE
made the extension "usable"

### PR DESCRIPTION
**Pull request by Samma2009**
it certanly isn't without bugs but it is better than before

### CODE REVIEW

File: JMC.Extension.Server/Handler/JMC/JMCCompletionHandler.cs [hunk 1]
```diff
@@ -95,9 +138,82 @@ private Task<CompletionList> GetJMCAsync(CompletionParams request, CancellationT
...
+                    Label = "for",
+                    Kind = CompletionItemKind.Snippet,
+                    InsertText = "for (confition) {\r\n}"
+                });
+                list.Add(new()
+                {
+                    Label = "switch",
+                    Kind = CompletionItemKind.Snippet,
+                    InsertText = "switch (convariable) {\r\ncase 1:\r\n say \"hi!\";}"
+                });
```
Comment (bdke) [inline JMC.Extension.Server/Handler/JMC/JMCCompletionHandler.cs]: make a list and foreach add them
it makes my ancestor suffering

File: JMC.Parser/JMC/Command/CommandParser.cs [hunk 2]
```diff
@@ -119,11 +119,12 @@ public IEnumerable<SyntaxNode> ParseIfCommand()
...
-            if (CurrentNode == null && CurrentChar != '\0')
-            {
-                Errors.Add(Index, "Expect End");
-            }
-            else if (CurrentNode?.Executable == null)
+            //if (CurrentNode == null && CurrentChar != '\0')
+            //{
+            //    Errors.Add(Index, "Expect End");
+            //}
+            //else
```
Comment (bdke) [inline JMC.Parser/JMC/Command/CommandParser.cs]: remove this

File: vscode-ext/.vscode/launch.json [hunk 3]
```diff
@@ -8,7 +8,6 @@
             "request": "launch",
             "runtimeExecutable": "${execPath}",
             "args": [
-                "--disable-extensions",
```
Comment (bdke) [inline vscode-ext/.vscode/launch.json]: reminder: use a vsocde profile that has jmc and spyglass extension only

**Review by bdke — CHANGES_REQUESTED** [review]
Good code quality
styles problem my issue

File: JMC.Parser/JMC/Command/CommandParser.cs (hunk 2 above)
Comment (SammaWasTaken) [reply JMC.Parser/JMC/Command/CommandParser.cs]: sure

File: vscode-ext/.vscode/launch.json (hunk 3 above)
Comment (SammaWasTaken) [reply vscode-ext/.vscode/launch.json]: yhea... i was too lazy to make it lol

**Review by bdke — APPROVED** [review]
Nice

**Pull request merged**